libafl/src/bolts/mod.rs

```diff
@@ -111,6 +110,22 @@ pub fn current_time() -> time::Duration {
     time::Duration::from_millis(millis)
 }
 
+/// Given a value v, return a random number using this mixing function
+/// Mixing function <http://mostlymangling.blogspot.com/2018/07/on-mixing-functions-in-fast-splittable.html>
```

Comment:
It may be a bit confusing what it's for with this name and placement 😬
Maybe at least note what it's used for in the comment (hashing an edge, or something)

Reply:
sure
nobody understands what `rrxmrrxmsx_0` means 😄

Reply:
Also, looking at this, there may be even better hashes for this use-case, interestingly 
https://github.com/martinus/better-faster-stronger-mixer#results suggests there may be even better options around.

Maybe we shoud link to the original doc, here https://mostlymangling.blogspot.com/2019/01/better-stronger-mixer-and-test-procedure.html

Reply:
Ah yes, already done, I'm slow :D

Reply:
Yeah add some info or move to a `hashes` mod, imho